gp-includes/formats/format_properties.php

```diff
@@ -0,0 +1,500 @@
+<?php
+
+class GP_Format_Properties extends GP_Format {
+
+	public $name = 'Java Properties File (.properties)';
+	public $extension = 'properties';
+	public $filename_pattern = '%s_%s';
+
+	public $exported = '';
+
+	/**
+	 * Generates a string the contains the $entries to export in the Properties file format.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param GP_Project $project The project the strings are being exported for, not used
+	 *                            in this format but part of the scaffold of the parent object.
+	 * @param GP_Locale $locale The locale object the strings are being exported for. not used
+	 *                          in this format but part of the scaffold of the parent object.
+	 * @param GP_Translation_Set $translation_set The locale object the strings are being 
+	 *                                            exported for. not used in this format but part
+	 *                                            of the scaffold of the parent object.
+	 * @param GP_Translation $entries The entries to export.
```

Comment:
That doesn't actually say they should be aligned to each other :wink:  

It just gives a poor example because they use the same type/variable name for both.

I'll align it :smile: